Quality checks

Sloy opened the pull request:
Added new formatting standard from Schibsted Spain, Checkstyle, PMD and Findbugs.

Sloy committed:
Added quality checks to the project

Checkstyle, PMD and Findbugs

Sloy committed:
Applied new formatting and fixed errors

Reformatted the project with the new code style, and fixed all the problems found by Checkstyle and PMD.

Sloy committed:
Added check task to travis process

CristianGM commented on .travis.yml:
```diff
@@ -27,7 +27,7 @@ android:
 install: /bin/true
 
-script: ./gradlew build
+script: ./gradlew check build
```
You don't need to call check, build already does it.

Sloy replied:
U sure? I know it does some checks, but I wasn't sure it ran the whole check task. I'll remove it then.

CristianGM commented on build.gradle:
```diff
@@ -1,24 +1,24 @@
-        // in the individual module build.gradle files
-    }
+    // NOTE: Do not place your application dependencies here; they belong
+    // in the individual module build.gradle files
```
would yo mind to clean this useless comment?

Sloy replied:
I will gladly do that, sir

CristianGM commented:
Your build failed, maybe you should take a look to this: http://stackoverflow.com/questions/35634437/travis-ci-build-failing-cause-permgen-space

Sloy committed:
Removed unwanted comments

Sloy committed:
Removed redudant check task

Build already runs check

Sloy committed:
Fix Travis OutOfMemory error

http://stackoverflow.com/questions/35634437/travis-ci-build-failing-cause-permgen-space
¯\_(シ)_/¯

Sloy commented:
Thanks @CristianGM, check it out now

CristianGM commented:
😍 👍
